.github/workflows/integration-test-iOS16_2.yaml

```diff
@@ -121,3 +121,11 @@ jobs:
           export PACKAGE_BRANCH_NAME=$GITHUB_HEAD_REF
           export PACKAGE_REVISION=$GITHUB_SHA
           swift test --package-path Examples/SPM -v
+      
+      - name: Carthage - Installation Test
```

Comment:
I don't think we need to run this three times (especially as it doesn't seem to vary by platform) — could we have a new workflow for it? It will also mean that it can run in parallel to the other build steps.

Reply:
https://github.com/ably/ably-cocoa/pull/1833/commits/9dba2a9770064d4aa5eba36d9f6ccd94e704bc1c